cycle2/empanelment/create_empanelment_lists.r

```diff
@@ -0,0 +1,98 @@
+# ngs2 cycle 2 empanelment lists
+# for questions, contact: matt hoover (matt_hoover@gallup.com)
+
+# clean environment and install packages that are required
+rm(list = ls())
+local <- TRUE # change to FALSE if getting data from the dod_clients drive
+
+# determine system environment and define constants for use
+if(Sys.info()['sysname'] == "Windows") {
+    dd <- "W:/DARPA_NGS2/CONSULTING/Analytics/cycle2/data"
```

Comment:
Replacing "W:/" with "//gallup/dod_clients" below works better for me:

dd <- "//gallup/dod_clients/DARPA_NGS2/CONSULTING/Analytics/cycle2/data"
od <- "//gallup/dod_clients/DARPA_NGS2/CONSULTING/Analytics/cycle2/output"